CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.5.0)
+project(cmake_common_scripts VERSION 0.1.0 LANGUAGES CXX)
+include(cmake/cmake_tools.cmake)
+include(cmake/code_coverage.cmake)
```

Comment:
Could you clarify why you `include` those snippets?

Shouldn't they be included by consumers of this project / get included as part of `find_package(..)`?

Reply:
They are not needed.